other/median_of_two_arrays.py

```diff
@@ -12,6 +27,9 @@ def find_median_arrays(nums1, nums2):
 
 
 if __name__ == "__main__":
-    array_1 = [int(x) for x in input("Enter the elements of first array: ").split()]
-    array_2 = [int(x) for x in input("Enter the elements of second array: ").split()]
-    print(f"The median of two arrays is: {find_median_arrays(array_1, array_2)}")
+    import doctest
+
+    doctest.testmod()
+    array_1: List[float] = [float(x) for x in input("Enter the elements of first array: ").split()]
+    array_2: List[float] = [float(x) for x in input("Enter the elements of second array: ").split()]
+    print(f"The median of two arrays is: {median_of_two_arrays(array_1, array_2)}")
```

Comment:
```suggestion
    array_1 = [float(x) for x in input("Enter the elements of first array: ").split()]
    array_2 = [float(x) for x in input("Enter the elements of second array: ").split()]
    print(f"The median of two arrays is: {median_of_two_arrays(array_1, array_2)}")

```
Python is smart enough to know that that list comprehension will generate a `List[float]` without our hint.  I tend to use type hints just for function parameters and return types to document these things for the caller of the funtion.